cmd/skupper/skupper_podman_site.go

```diff
@@ -78,7 +79,26 @@ func (s *SkupperPodmanSite) Create(cmd *cobra.Command, args []string) error {
 	if routerCreateOpts.Ingress != types.IngressNoneString {
 		// Validating ingress hosts (required as certificates must have valid hosts)
 		if len(site.IngressHosts) == 0 {
-			return fmt.Errorf("At least one ingress host is required")
+			// Get all unicast IP addresses on the machine
+			addresses, err := net.InterfaceAddrs()
+			if err != nil {
+				return fmt.Errorf("Cannot get a default ingress host")
+			}
+			var ingressHosts []string
+			for _, address := range addresses {
+				ipnet, ok := address.(*net.IPNet)
+				if ok && !ipnet.IP.IsLoopback() && ipnet.IP.To4() != nil {
+					ipv4ValidAddress := ipnet.IP.String()
```

Comment:
I believe that the `ipv4ValidAddress` could always be added to the `ingressHosts` slice.